torchtune/modules/low_precision/_register_nf4_dispatch_ops.py

```diff
@@ -6,7 +6,6 @@
 
 import torch
 from torchao.dtypes.nf4tensor import implements as nf4_tensor_impl, to_nf4
-from torchtune.modules.low_precision._utils import _get_torchao_version
```

Comment:
Strictly speaking this util is now no longer used anywhere in our library, but I am inclined to keep it in for possible (likely?) future usage

Reply:
Nope - get rid of it until we need it again. It's easy enough to find in git history.